### ISSUE TITLE
Appeal process

### ISSUE THREAD

**Issue by ohofmann** [ga4gh/product-process#13]
Looking at the flowchart and the accompanying description there are a number of steps left to the secretariat and TASC - has enough work been done to identify stakeholders, is this of sufficient interest to the community, which PRC members should be drafted, etc.

It might be helpful to think about what an appeal process might look like. Is there any way for someone to bring a new (declined) project proposal to the attention of the steering committee or another entity? I realise we don't want anyone to abuse the process, and much as an appeal to the editor after a declined paper submission usually has a _very_ slim chance of succeeding I feel it would be helpful to have some process in place or to clarify that decisions of the secretariat and TASC are to be considered final.

**Comment by vdauwera** [comment]
Strongly second this — In general I would love to see more clarity around what amount of feedback/justification would be given to a rejected proposal (beyond "the SC voted no"), and what would be the conditions for requesting a review of the decision (one could imagine the proposers might request a specific type of review -- eg technical, policy, or EDI-based)

**Comment by michaelmhoffman** [comment]
To clarify: the request here is not for appeals of decisions by the Steering Committee, right? (That is a whole other ball of wax.)

**Comment by ohofmann** [comment]
> To clarify: the request here is not for appeals of decisions by the Steering Committee, right? (That is a whole other ball of wax.)

No, I was thinking of the submitter here.

**Comment by jmarshall** [comment]
:+1: Very much agree with @vdauwera here. The technical parts of GA4GH have to date largely operated by consensus — which incidentally is in line with the principles mentioned in #1 which we feel we are broadly aligned with.

To fit in with this model, rejection decisions at any of these steps should be expected to be usually in the form of “[detailed list of items to be addressed in order for this proposal to be accepted]”.

**Comment by michaelmhoffman** [comment]
@ohofmann I mean you are looking for appeals of decisions made by e.g. Secretariat and TASC and not appeals of decisions made by the Steering Committee?

**Comment by ohofmann** [comment]
> @ohofmann I mean you are looking for appeals of decisions made by e.g. Secretariat and TASC and not appeals of decisions made by the Steering Committee?

Correct. Do you feel there should be a separate process for the SC decisions? In the past, proposals and standards had usually processed far enough that a negative vote by the SC usually meant 'come back with the following fixes', not an outright rejection.

**Comment by michaelmhoffman** [comment]
I think that appeals of SC decisions are YAGNI, and it's super unclear how it would work.

**Comment by susanfairley** [comment]
Thanks for the comments here.

One clarification: TASC doesn't have a decision making role in this process. TASC does have responsibility for technical alignment decisions at the GA4GH/organisational level. The question of if a standard complies with GA4GH technical alignment policies is reviewed by the PRC (and anyone else who wants to weigh in on public comment for that matter).

For feedback to the developers, one of the aims of this process is to, effectively, compel consultation before and during development. That includes interested members of SC. It is aiming to make it very, very difficult for feedback not to have been heard well before things reach an SC vote. Happy to hear suggestions on changes needed to make sure that that happens. Again, one of the aims of this is to make sure that SC opinions (and those from elsewhere) are heard early in the process, so any issues related to obtaining that feedback need to be flagged. This also relates to keeping those who are interested informed as the work progresses, to facilitate further feedback.

I also think consensus is important. If that isn't apparent in the document, then that is a concern.

The decision makers are: Secretariat, the subgroup (or people trying to get a project off the ground), the PRC and SC.

For SC, bluntly, if the consultation has been done, there shouldn't be anything new to hear by the time SC members make a decision and vote. I'm open to considering an appeal option but it's the responsibility of the developers (using the support from Secretariat) to have sought opinion widely well before the SC vote. That's the intended feedback mechanism. It probably ought not to progress to SC until that consensus has been built.

For Secretariat decisions, there is an appeal process - take it to exec who can in turn take it to SC.

For group/subgroup decisions, things get much more varied. The decision making process (and appeal) is up to the subgroup but it is being stipulated here that whatever their process is, it needs to be written down.

For the PRC, they are intended to review against the criteria. The intention here is that this should be a dialogue between the PRC and subgroup, however, I can imagine a PRC voting against a piece of work. If the PRC aren't persuaded, the decision on next steps falls back to the subgroup and Secretariat. The hope would be that this discussion is focussed on does the standard meet the approval criteria and is constructive. That should ensure that feedback is given. I'm probably more supportive of an appeal mechanism here but would have some concerns about how we find people with sufficient domain knowledge - finding PRC members can be challenging. We could look at referring to exec? As the process is currently described, the subgroup consider what they want to do with a final decision resting with Secretariat (which is appealable).

Further thoughts: the PRC have an important role in this. There is also #21 on PRC composition. Voting arrangements are primarily up for discussion on #6. Finally, the role of Secretariat #23.

Further comments would be welcome.

**Comment by susanfairley** [comment]
Some changes made which relate to this discussion.

If the study group outputs are rejected, the group gets to decide next steps and can come back for a second time. This is also the case at 19, where groups consider options after work has been rejected at later stages.

The vote barrier has gone up to 60% of cast votes in favour, to demonstrate consensus, but the opportunity to come back is now made explicit. In addition, the voting form is amended to allow feedback at the time of votes on content.

The option to use the dispute resolution mechanism in the event of people being unhappy with a decision is also made explicit.

I hope that these changes improve clarity on these issues. Link to the commit is above.

Thanks again for the feedback.
